restapi/endpoints_test.go

```diff
@@ -30,9 +29,9 @@ func (suite *endpointTestSuite) SetupSuite() {
 	suite.db = db
 
 	// setup required for jwt authentication
-	viper.SetDefault("auth.jwt_key", "random")
-	viper.SetDefault("auth.jwt_expiry", "5m")
-	viper.SetDefault("auth.jwt_issuer", "endpoint_tests")
+	viper.SetDefault("auth.jwt_key", testutils.JWTKey)
```

Comment:
are these from the .env file in the regular app? If so can we add them to heroku as well

Reply:
no, these are from `testutils` file; the tests are written in isolation from rest of app, so we need to set env variables. i do this in the testutils file

Reply:
okay as long as you have added any new env variables to heroku should be fine